image/templates/helm/stackrox-secured-cluster/templates/_admission-controller-config.tpl

```diff
@@ -0,0 +1,19 @@
+{{- define "srox.protectAdmissionControllerConfig" -}}
+{{- $ := . -}}
+
+{{- $formatMsg := "It is not supported anymore to specify 'admissionControl.%s'. This setting will be ignored." -}}
```

Comment:
As a user I'd be concerned seeing that my setting will be ignored, if I set one of these to "true". Perhaps something reassuring such as the following would help?

```suggestion
{{- $formatMsg := "It is not supported anymore to specify 'admissionControl.%s'. This setting will be ignored. The effective value is 'true'." -}}
```